index.html

```diff
@@ -21,7 +21,7 @@
 <!-- load functions & object definitions that we'll call later -->
 		<script src='scripts/functions.js'></script>
 
-<!-- jQuery UI.  A custom build of it including only the tabs widget and dependencies of that -->
+<!-- jQuery UI.  A custom build of it including only the tabs and dialog widgets and dependencies of those -->
```

Comment:
All the changes in the ` jquery-ui-1.13.0.custom` folder are reflections of the above